cmd/kube-apiserver/app/server.go

```diff
@@ -134,11 +136,17 @@ func CreateServerChain(runOptions *options.ServerRunOptions, stopCh <-chan struc
 	if err != nil {
 		return nil, err
 	}
+	groups, err := resourceconfig.ParseGroups(runOptions.APIEnablement.RuntimeConfig)
```

Comment:
how about moving all these calls to one place? maybe easier to follow.

Reply:
make sense.